Lecture3/Task1/product/ProductFactoryDirector.java

```diff
@@ -0,0 +1,44 @@
+package product;
+
+import builder.AcousticGuitarBuilder;
+import builder.ElectricGuitarBuilder;
+import builder.Guitar;
+import builder.GuitarBuilderDirector;
+import exceptions.FailedOrderException;
+import utils.RandomUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductFactoryDirector {
```

Comment:
this class in responsible for building guitars- so please rename it to GuitarFactory

Reply:
done